codec-http2/src/test/java/io/netty/handler/codec/http2/Http2CodecTest.java

```diff
@@ -110,10 +111,10 @@ public void createOutboundStream() {
         Http2Headers headers = new DefaultHttp2Headers();
         childChannel.write(new DefaultHttp2HeadersFrame(headers));
         ByteBuf data = Unpooled.buffer(100).writeZero(100);
-        childChannel.writeAndFlush(new DefaultHttp2DataFrame(data));
+        childChannel.writeAndFlush(ReferenceCountUtil.releaseLater(new DefaultHttp2DataFrame(data)));
```

Comment:
The stream channel does the https://github.com/netty/netty/blob/4.1/codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamChannel.java#L183

At the end of this test, the reference count is 1, which seems legit to me? Will be set to 0 by `releaseLater`.